src/erf.jl

```diff
@@ -413,7 +415,55 @@ function erfcinv(y::Float32)
     end
 end
 
-erfcinv(x::Integer) = erfcinv(float(x))
+function erfinv(y::BigFloat)
+    xfloat = erfinv(Float64(y))
+    sqrtπ = sqrt(big(pi))
+    if isfinite(xfloat)
+        x = BigFloat(xfloat)
+    else
+        # Float64 overflowed, use asymptotic estimate instead
+        # from erfc(x) ≈ exp(-x²)/x√π ≈ y  ⟹  -log(yπ) ≈ x² + log(x) ≈ x²
+        x = copysign(sqrt(-log((1-abs(y))*sqrtπ)), y)
+    end
+    isfinite(x) || return x
+    sqrtπhalf = sqrtπ * 0.5
+    tol = 2eps(abs(x))
+    while true # Newton iterations
+        Δx = sqrtπhalf * (erf(x) - y) * exp(x^2)
+        x -= Δx
+        abs(Δx) < tol && break
+    end
+    return x
+end
+
+function erfcinv(y::BigFloat)
+    yfloat = Float64(y)
+    xfloat = erfcinv(yfloat)
+    sqrtπ = sqrt(big(pi))
+    if isfinite(xfloat)
+        x = BigFloat(xfloat)
+    else
+        # Float64 overflowed, use asymptotic estimate instead
+        # from erfc(x) ≈ exp(-x²)/x√π ≈ y  ⟹  -log(yπ) ≈ x² + log(x) ≈ x²
+        if yfloat < 1
+            x = sqrt(-log(y*sqrtπ))
+        else # y must be close to 2
+            @show x = -sqrt(-log((2-y)*sqrtπ))
```

Comment:
```suggestion
            x = -sqrt(-log((2-y)*sqrtπ))
```

Reply:
Whoops, sorry

Reply:
Fixed in 4e8836c55eacae2f8d063f9d0702f741a1376211